cypress.config.ts

```diff
@@ -0,0 +1,45 @@
+import { defineConfig } from 'cypress';
+require('dotenv').config();
+
+/**
+ * Filter test spec paths based on env var configuration
+ * @returns
+ */
+const getSpecPattern = (): string[] => {
```

Comment:
Why does this need to be a function?

Reply:
Because there's a bunch of computations and there's no need to keep them exposed outside the function.